packages/types/README.md

```diff
@@ -1,4 +1,25 @@
 # @cennznet/types
+[27/03/2020]  1.0.0 version:
+
+ - `@cennznet/api@1.0.0` (current stage) is working with [cennznet/cennznet:latest][cennznet/cennznet:latest];
```

Comment:
Just need to update type related changes if any, otherwise  no need to repeat api change notes :)

Reply:
done